forgerock-auth/src/main/java/org/forgerock/android/auth/OAuth2Client.kt

```diff
@@ -154,7 +158,7 @@ class OAuth2Client(
      * @param accessToken The AccessToken to be revoked
      * @param listener    Listener to listen for revoke event
      */
-    fun revoke(accessToken: AccessToken, listener: FRListener<Void?>?) {
+    fun revoke(accessToken: AccessToken, listener: FRListener<Void?>) {
```

Comment:
Would this change to the second parameter affect the backward compatibility?

Reply:
The interface is for internal use only, I don't think any developer is using this interface. And I have confirmed that all caller in the SDK pass a non null object.